index.html

```diff
@@ -57,14 +58,10 @@ <h4>Zero-ops OpenStack on just about any Linux box.</h4>
     <div class="row">
       <div class="col-8">
         <p>
-          A full OpenStack in a single snap package! MicroStack is an upstream multi-node OpenStack deployment which can
-          run directly on your workstation. Although made for developers, it is also suitable for edge, IoT and
-          appliances. Do not wait anymore - grab MicroStack from the Snap Store and get your OpenStack running right
-          away!
+          A full OpenStack in a single snap package. MicroStack is an upstream multi-node OpenStack deployment which can run directly on your workstation. Although made for developers, it is also suitable for edge, IoT and appliances. Grab MicroStack from the Snap Store and get your OpenStack running right away.
         </p>
         <p>
-          <a class="p-link--external" href="https://tutorials.ubuntu.com/tutorial/microstack-get-started#0">Get started
-            with MicroStack tutorial</a>
+          <a class="p-link--external" href="https://tutorials.ubuntu.com/tutorial/microstack-get-started#0">Get started with MicroStack tutorial</a>
```

Comment:
Could you update the link to https://ubuntu.com/tutorials/microstack-get-started#1-overview

Reply:
done.